### PR TITLE
Several NuGet curations

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: curations/NuGet/Microsoft.Net/Http.Headers.yml [hunk 1]
```diff
@@ -1,5 +1,10 @@
-- id: "NuGet:Microsoft.Net:Http.Headers:[2.1.7,)"
```
[the remark this answers is not in the document]
Comment (fviernau) [reply curations/NuGet/Microsoft.Net/Http.Headers.yml:1]: The syntax is documented here: https://ant.apache.org/ivy/history/2.1.0/settings/version-matchers.html

Yes, the previous curation was wrong.

File: curations/NuGet/Microsoft.AspNetCore/Hosting.Abstractions.yml [hunk 2]
```diff
@@ -1,5 +1,10 @@
-- id: "NuGet:Microsoft.AspNetCore:Hosting.Abstractions:[2.1.7,)"
```
[the remark this answers is not in the document]
Comment (fviernau) [reply curations/NuGet/Microsoft.AspNetCore/Hosting.Abstractions.yml:1]: Yes it was.

**Review by haikoschol — APPROVED** [review]

File: curations/NuGet/_/log4net.yml [hunk 3]
```diff
@@ -1,39 +1,46 @@
 - id: "NuGet::log4net:2.0.8"
   curations:
-    comment: "Set the VCS revision because there are multiple tags matching the version."
+    comment: "Multiple tags matching this version.""
```
Comment (haikoschol) [inline curations/NuGet/_/log4net.yml:3]: There's a duplicate `"` at the end.

File: curations/NuGet/_/log4net.yml [hunk 4]
```diff
@@ -1,39 +1,46 @@
...
     vcs:
-      type: Git
-      url: "https://github.com/apache/logging-log4net.git"
       revision: "rel/2.0.8"
 - id: "NuGet::log4net:2.0.9"
   curations:
-    comment: "Set the VCS revision because there are multiple tags matching the version."
+    comment: "Multiple tags matching this version.""
```
Comment (haikoschol) [inline curations/NuGet/_/log4net.yml:8]: There's a duplicate `"` at the end.

File: curations/NuGet/_/log4net.yml [hunk 5]
```diff
@@ -1,39 +1,46 @@
...
     vcs:
-      type: Git
-      url: "https://github.com/apache/logging-log4net.git"
       revision: "rel/2.0.9"
 - id: "NuGet::log4net:2.0.10"
   curations:
-    comment: "Set the VCS revision because there are multiple tags matching the version."
+    comment: "Multiple tags matching this version.""
```
Comment (haikoschol) [inline curations/NuGet/_/log4net.yml:13]: There's a duplicate `"` at the end.

File: curations/NuGet/_/log4net.yml [hunk 6]
```diff
@@ -1,39 +1,46 @@
...
     vcs:
       revision: "rel/2.0.10"
 - id: "NuGet::log4net:2.0.11"
   curations:
-    comment: "Set the VCS revision because there are multiple tags matching the version."
+    comment: "Multiple tags matching this version.""
```
Comment (haikoschol) [inline curations/NuGet/_/log4net.yml:18]: There's a duplicate `"` at the end.

File: curations/NuGet/_/log4net.yml [hunk 7]
```diff
@@ -1,39 +1,46 @@
...
     vcs:
       revision: "rel/2.0.11"
 - id: "NuGet::log4net:2.0.12"
   curations:
-    comment: "Set the VCS revision because there are multiple tags matching the version."
+    comment: "Multiple tags matching this version.""
```
Comment (haikoschol) [inline curations/NuGet/_/log4net.yml:23]: There's a duplicate `"` at the end.

File: curations/NuGet/_/log4net.yml [hunk 8]
```diff
@@ -1,39 +1,46 @@
...
     vcs:
       revision: "rel/2.0.12"
 - id: "NuGet::log4net:2.0.13"
   curations:
-    comment: "Set the VCS revision because there are multiple tags matching the version."
+    comment: "Multiple tags matching this version.""
```
Comment (haikoschol) [inline curations/NuGet/_/log4net.yml:28]: There's a duplicate `"` at the end.

File: curations/NuGet/_/log4net.yml [hunk 9]
```diff
@@ -1,39 +1,46 @@
...
     vcs:
       revision: "rel/2.0.13"
 - id: "NuGet::log4net:2.0.14"
   curations:
-    comment: "Set the VCS revision because there are multiple tags matching the version."
+    comment: "Multiple tags matching this version.""
```
Comment (haikoschol) [inline curations/NuGet/_/log4net.yml:33]: There's a duplicate `"` at the end.

**Review by haikoschol — CHANGES_REQUESTED** [review]

**Review by haikoschol — APPROVED** [review]

**Review by haikoschol — APPROVED** [review]

**Comment by fviernau** [comment]
I've integrated the changes into https://github.com/oss-review-toolkit/ort-config/pull/225.

**Pull request closed without merging**